### DISCUSSION TITLE
I applied for the GitHub Education benefits at 11 Aug, it got rejected twice since then. I reapplied at 18 Aug and it shows approved, but also Congrats! Your request was approved and your benefits will be available soon.

### DISCUSSION THREAD

**Discussion by Ywatch15** [community/community#136352] [GitHub Education]
What should I do now? How long do I have to wait?

**Comment by master-coco** [comment]
Hey 
Congratulations on getting your GitHub Education benefits approved! Here's what you should do next:

Check Your Email: GitHub Education typically sends an email confirming that your benefits are available. Look for any instructions or further steps in the email.

Visit GitHub Education: Log in to your GitHub Education account and check your dashboard for any updates or additional instructions. Sometimes, there might be a link to activate or claim your benefits.

Explore Your Benefits: Once your benefits are available, you can start exploring the various perks. GitHub Education usually includes free access to certain tools, discounts, and other resources for students.

Wait Time: The time it takes for benefits to become available can vary. It might be immediate, but sometimes it could take a few days for everything to process fully. If you don’t see your benefits within a week, it might be worth reaching out to GitHub Education support for an update.

Check for Updates: Make sure your GitHub account and any linked educational accounts are properly set up and verified.

If you run into any issues or have questions, GitHub Education's support team is there to help. Good luck with your studies and projects!

**Reply by queenofcorgis** [reply]
@master-coco, 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by master-coco** [comment]
talking from personal experience though, mine took minutes last year but weeks this year 
They might have just changed the processing time or backend stuff which is causing the delay so best bet would be to wait it out and email them if it takes too long

**Comment by davevad93** [comment]
Hi @Ywatch15 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Ywatch15 was approved and your benefits are now available.`

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Ywatch15 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
